modin/pandas/groupby.py

```diff
@@ -815,36 +822,49 @@ def _index_grouped(self):
         What this returns (if anything)
         """
         if self._index_grouped_cache is None:
-            if hasattr(self._by, "columns") and len(self._by.columns) > 1:
+            is_multi_by = self._is_multi_by or (
```

Comment:
`is_multi_by = False` in context of this function means: get groups by grouping on index
https://github.com/modin-project/modin/blob/cb5aa4f2a81d9df2b4e11b3e7f8ebec5fa8265eb/modin/pandas/groupby.py#L850-L853
`pandas.Index.groupby` can obtain either 1d array or MultiIndex, so no ability to process multiple `by`s.